### ISSUE TITLE
Any ETA for exposing the User Messaging Platform API?

### ISSUE THREAD

**Issue by NickCullen** [googleads/googleads-mobile-unity#2257]
Hey,

Reading the AdMob documentation, it seems like the new [User Messaging Platform SDK](https://support.google.com/admanager/answer/10113209?hl=en) is included within the Google Mobile Ads library

However, I can't see any C# bindings for this functionality. 

Is there plans to include this any time soon?

**Comment by NVentimiglia** [comment]
Hi @NickCullen,

I can not give an exact release date, but this feature is currently being worked on.

**Comment by binouze** [comment]
Hi, @NickCullen if you still need it I just put a simple implementation of the UMP SDK for Unity [here](https://github.com/binouze/GoogleUserMessagingPlatform)

**Comment by NickCullen** [comment]
Hi @binouze 

thanks, I ended up doing the same natively. I got it working on iOS but then when I went to do Android I got an email from AdMob saying they reviewed my app (even though there was no build they could have possibly tested with) and declined me from the program. Can no longer log in to that AdMob account now. Seems to be a common issue looking online….

Because of this, and no support contact for AdMob, I went with Applovin in the end.
